website/source/docs/guides/connect-production.md

```diff
@@ -52,50 +53,25 @@ words, any client that has permission to _register_ an instance of a service
+    must be `deny`.** If for any reason you cannot use the default policy of
+    `deny`, you must add an explicit ACL denying anonymous `service:write`. Note, in this case the Connect intention graph will also default to
     `allow` and explicit `deny` intentions will be needed to restrict service
     access. Also note that explicit rules to limit who can manage intentions are
     necessary in this case. It is assumed for the remainder of this guide that
     ACL policy defaults to `deny`.
- 2. **Each service must have a distinct ACL token** that is restricted to
-    `service:write` only for the named service. Current Consul ACLs only support
-    prefix matching but in a near-future release we will allow exact name
-    matching. It is possible for all instances of the service to share the same
-    token although best practices is for each instance to get a unique token as
-    described below.
-
-### Fine Grained Enforcement
+ 2. **Each service must have a unique ACL token** that is restricted to
+    `service:write` only for the named service. You can review the [Securing Consul with ACLs](https://learn.hashicorp.com/consul/advanced/day-1-operations/production-acls#apply-individual-tokens-to-the-services) guide for a 
+    service token example. Note, it is best practices for each instance to get a unique token as described below.
 
-Connect intentions manage access based only on service identity so it is
-sufficient for ACL tokens to only be unique per _service_ and shared between
+~> Individual Service Tokens: It is best practice to create a unique ACL token per service _instance_ because
+it limits the blast radius of a compromise. However, since Connect intentions manage access based only on service identity, it is
+possible to create only one ACL token per _service_ and share it between
 instances.
 
-It is much better though if ACL tokens are unique per service _instance_ because
-it limits the blast radius of a compromise.
-
-A future release of Connect will support revoking specific certificates that
-have been issued. For example if a single node in a datacenter has been
-compromised, it will be possible to find all certificates issued to the agent on
-that node and revoke them. This will block all access to the intruder without
-taking instances of the service(s) on other nodes offline too.
-
-While this will work with service-unique tokens, there is nothing stopping an
-attacker from obtaining certificates while spoofing the agent ID or other
-identifier – these certificates will not appear to have been issued to the
-compromised agent and so will not be revoked.
-
-If every service instance has a unique token however, it will be possible to
-revoke all certificates that were requested under that token. Assuming the
-attacker can only access the tokens present on the compromised host, this
-guarantees that any certificate they might have access to or requested directly
-will be revoked.
-
```

Comment:
I think some of this was good material but in practice no one uses this model and it will be a while before it matters much. By the time it is we will likely have better ways to manage service identity like R.B.'s JWT stuff so I'm fine with dropping it for now.